source/slang/slang-check-modifier.cpp

```diff
@@ -474,6 +474,17 @@ namespace Slang
             }
             overloadRankAttr->rank = int32_t(rank->getValue());
         }
+        else if (auto inputAttachmentIndexLayoutAttribute = as<GLSLInputAttachmentIndexLayoutAttribute>(attr))
+        {
+            if (attr->args.getCount() != 1)
```

Comment:
https://github.com/shader-slang/slang/blob/master/docs/design/coding-conventions.md suggests no space between if and (

Reply:
I genuinely don't know if we are following that style suggestion, it seems for the last 5(6?) years of code in this file the suggestion has never been followed.

Should I start following it...?

Reply:
I think we need to update the document for the spacing in the control flow.
Because I don't think we are doing it.

Reply:
> I think we need to update the document for the spacing in the control flow.
Because I don't think we are doing it.

I agree, likely best since this part of the styling is not being (has never been?) followed.

Reply:
I created a PR for the document update.
https://github.com/shader-slang/slang/pull/4498

Reply:
I wouldn't mind if somebody just run clang format through the code base and we stick to whatever that format is from that point on.